workflow/scripts/ts_processors.py

```diff
@@ -198,110 +198,63 @@ class tskit_windowed_sfs_plus_ld(BaseProcessor):
     def __init__(self, config: dict):
         super().__init__(config, self.default_config)
         
-    # TODO: this was copied from Yuxin's code, need to clean it up.
-    # not sure if we need the circular option at all?
-    def _LD(self, haplotype, pos_vec, size_chr, circular=True, distance_bins=None):
+    # Cleaned up LD calculator
+    def _LD(haplotype, pos_vec, size_chr, distance_bins=None):
         """
-        Compute LD for a subset of SNPs drawn with different gap sizes in between them.
-        Gap sizes follow power 2 distribution.
-        The LD is then computed and averaged over different bin (distance_bins) sizes.
-
-        Parameters ---------- haplotype : numpy 2D array or allel.haplotype SNP matrix where in the first dimension are
-        the SNP (rows) and in the second dimension (columns) are the samples. pos_vec : 1D array array of absolute
-        positions in [0, size_chr]. size_chr : int Size of the chromosome. circular : bool Whether to consider the
-        chromosome circular or not. If circular, the maximum distance between 2 SNPs is thus half the chromosome.
-        distance_bins : int or list LD will be averaged by bins of distances e.g. if distance_bins = [0, 100, 1000,
-        10000], LD will be averaged for the groups [0,100[, [100, 1000[, and [1000, 10000[ If distance_bins is an int,
-        it defines the number of bins of distances for which to compute the LD The bins are created in a logspace If
-        distance_bins is a list, they will be used instead
-
-        Returns
-        -------
-        DataFrame
-            Table with the distance_bins as index, and the mean value of
+        Compute LD for a subset of SNPs and return a DataFrame with only the mean r2.
```

Comment:
i think it's useful to keep some of the details about the parameters that are needed in here